lifecycle/lifecycle-viewmodel-compose/src/commonMain/kotlin/androidx/lifecycle/viewmodel/compose/SavedStateHandleSaver.kt

```diff
@@ -119,7 +119,7 @@ fun <T : Any> SavedStateHandle.saveable(
     init: () -> T,
 ): PropertyDelegateProvider<Any?, ReadOnlyProperty<Any?, T>> =
     PropertyDelegateProvider { thisRef, property ->
-        val classNamePrefix = if (thisRef != null) thisRef::class.canonicalName + "." else ""
```

Comment:
This should be `canonicalName`. During `jb-main` merge to `integration-releaser/lifecycle/...` there should be a conflict, in which `canonicalName` should be chosen (I should did it myself in the past when merged 2.8.4, sorry 🫠)

Reply:
Also, please, push `integration-release/lifecycle/2.8`

Reply:
done

Reply:
After merging the PR, also merge `jb-main` again to `integration-release/lifecycle/2.8`